KlingonAssistant/data/mem-03-D.xml

```diff
@@ -438,7 +438,7 @@ As a time element, {DaHjaj:n} precedes any adverbials.[1, sec. 6.7]</column>
       <column name="entry_name">Damu'</column>
       <column name="part_of_speech">n</column>
       <column name="definition">crease</column>
-      <column name="definition_de"></column>
+      <column name="definition_de">Falte (im Papier)</column>
```

Comment:
It can mean wrinkle, the lines you have in your face. Of course, it may mean the same, we should ask Maltz.
What I'm also not sure is wheather "crease" is what you get after folding a paper, or also when you crash apiece of paper to a ball and then unfold it. Those are creases as well, right?